docs/data-ingestion/Proxmox.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Proxmox
+weight: 1
+sort: 1
+menu:
+  docs:
+    identifier: "proxmox"
+    parent: "data-ingestion"
+    weight: 1
+aliases:
+  - /data-ingestion/proxmox.html
+  - /data-ingestion/Proxmox.html
+---
+
+Proxmox Virtual Environment(PVE) and Proxmox Backup Server(PBS) support sending data using the InfluxDB protocol. So we can use the InfluxDB write support built into VictoriaMetrics.
+Currently, PVE and PBS only support using an Authorization Token for authentication and does not support basic auth or a username and password.
+
+## Proxmox Virtual Environment (PVE)
+If you want help sending data to Managed VictoriaMetrics from PVE check out [this blog post](https://victoriametrics.com/blog/proxmox-monitoring-with-dbaas/).
+
+1. Login to PVE as an administrator
+2. Go to DataCenter > MetricServer > Add > InfluxDB
+3. Set the parameters as follows:
+  - Name: VictoriaMetrics (can be changed to any string)
+  - Server: the hostname or IP of your VictoriaMetrics Instance
+  - Port: This will vary depending how you are sending data to VictoriaMetrics, but these are the defaults for the various components of VictoriaMetrics.
+    - 8428 for single-node VictoriaMetrics
+    - 8480 for vminsert
+    - 8427 for vmauth
+    - 8429 for vmagent
```

Comment:
I find it inconsistent to list all the ports here, and omitting them in the rest examples. 
Shall we list these ports once somewhere and refer to them in other docs? wdyt @tiny-pangolin @qphuongl ?

Reply:
I agree maybe we can add something in the `_index.md` that lists the default port for all the components?

Reply:
I will note this down and resolve it later. The ports and general information of these components will be explained in the main document